tests/pytests/test_config.py

```diff
@@ -75,23 +76,26 @@ def testSetConfigOptions(env):
     env.expect('ft.config', 'set', 'MAXEXPANSIONS', 1).equal('OK')
     env.expect('ft.config', 'set', 'TIMEOUT', 1).equal('OK')
     if MT_BUILD:
-        env.expect('ft.config', 'set', 'WORKER_THREADS', 1).equal(not_modifiable)
+        env.expect('ft.config', 'set', 'WORKER_THREADS', 1).equal('OK')
     env.expect('ft.config', 'set', 'FRISOINI', 1).equal(not_modifiable)
-    env.expect('ft.config', 'set', 'ON_TIMEOUT', 1).equal('Success (not an error)')
+    env.expect('ft.config', 'set', 'ON_TIMEOUT', 1).equal('Invalid ON_TIMEOUT value')
     env.expect('ft.config', 'set', 'GCSCANSIZE', 1).equal('OK')
     env.expect('ft.config', 'set', 'MIN_PHONETIC_TERM_LEN', 1).equal('OK')
     env.expect('ft.config', 'set', 'GC_POLICY', 1).equal(not_modifiable)
     env.expect('ft.config', 'set', 'FORK_GC_RUN_INTERVAL', 1).equal('OK')
     env.expect('ft.config', 'set', 'FORK_GC_CLEAN_THRESHOLD', 1).equal('OK')
     env.expect('ft.config', 'set', 'FORK_GC_RETRY_INTERVAL', 1).equal('OK')
 
+@skip(cluster=True)
 def testSetConfigOptionsErrors(env):
     env.expect('ft.config', 'set', 'MAXDOCTABLESIZE', 'str').equal(not_modifiable)
-    env.expect('ft.config', 'set', 'MAXEXPANSIONS', 'str').equal('Success (not an error)')
-    env.expect('ft.config', 'set', 'TIMEOUT', 'str').equal('Success (not an error)')
-    env.expect('ft.config', 'set', 'FORKGC_SLEEP_BEFORE_EXIT', 'str').equal('Success (not an error)')
-    env.expect('ft.config', 'set', 'FORKGC_SLEEP_BEFORE_EXIT', 'str').equal('Success (not an error)')
-'''
+    env.expect('ft.config', 'set', 'MAXEXPANSIONS', 'str').equal('Could not convert argument to expected type')
+    env.expect('ft.config', 'set', 'TIMEOUT', 'str').equal('Could not convert argument to expected type')
+    env.expect('ft.config', 'set', 'FORKGC_SLEEP_BEFORE_EXIT', 'str').equal('Could not convert argument to expected type')
+    env.expect('ft.config', 'set', 'FORKGC_SLEEP_BEFORE_EXIT', 'str').equal('Could not convert argument to expected type')
+    if MT_BUILD:
+        env.expect('ft.config', 'set', 'WORKER_THREADS',  2 ** 13 + 1).contains('Number of worker threads cannot exceed')
```

Comment:
invalid MT_MODE can be covered here as well, your choice.

Reply:
It's immutable so there is no point. We have another test for immutables